README.md

```diff
@@ -74,16 +80,41 @@ with **at least** the following roles:
   - Get and list access to GKE Clusters.
 `
 
-### Via Credentials
+### Via google-github-actions/auth
+
+You can authenticate use [google-github-actions/auth](https://github.com/google-github-actions/auth) directly to the action
+by specifying the `credentials` input. This Action supports both the recommended [Workload Identity Federation][wif]
+based authentication and the traditional [Service Account Key JSON][sa] based auth.
+
+See [usage](https://github.com/google-github-actions/auth#usage) for more details.
+
+#### Authenticating via Workload Identity Federation
+
+```yaml
+- id: auth
+  uses: google-github-actions/auth@v0.4.0
+  with:
+    workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
+    service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
+    create_credentials_file: true
```

Comment:
I'd omit all of the "create_credentials_file" options as it's the default.

Reply:
done